src/Math-Tests-Complex/PMComplexTest.class.st

```diff
@@ -783,6 +729,72 @@ PMComplexTest >> testTanh [
 	self assert: (c2 imaginary closeTo: c tanh imaginary).
 ]
 
+{ #category : #'testing - complex conjugation' }
+PMComplexTest >> testThatComplexConjugateHasEqualRealPartAndAnImaginaryPartWithEqualMagnitudeButOppositeSign [
```

Comment:
Those long test names are very hard to read even if I can see all of it (and System Browser will hide most of the name because of the window size).
```
testThatComplexConjugateHasEqualRealPartAndAnImaginaryPartWithEqualMagnitudeButOppositeSign
```
I really prefer this name:
```
testComplexConjugate
```
Why?
Because we are testing if `complexConjugate` method returns the correct complex conjugate.
That's it.
Saying that _"it must have equal real part and an imaginary part with equal magnitude but opposite sign"_ is just the definition of a complex conjugate. This sentence should be in a documentation comment of the `complexConjugate` method but not in the test name

Reply:
There is a bit more to complex conjugation e.g. it is distributive over the binary operations for example hence:

I would say `testWeCanComputeTheComplexConjugate` 

but if you insist I'm happy to go with what you suggest.

Reply:
Also, there is the value or idea of **Tests as documentation** c.f. XUnit Test Patterns by Géerard Mezsaros